templates/home/index.html.twig

```diff
@@ -1,7 +1,55 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}Projektai{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for dataKey in dataSet|keys %}
```

Comment:
Twig palaiko `key, value` formatą

Plačiau:
 * https://twig.symfony.com/doc/2.x/tags/for.html#iterating-over-keys-and-values